src/menu.pl

```diff
@@ -23,7 +23,8 @@
     write('Escolha o modo de jogo:\n'),
     write('[1] - Multiplayer Local\n'),
     write('[2] - Multiplayer Socket\n'),
-    write('[3] - Computador\n'),
+    write('[3] - Computador Local\n'),
+    write('[4] - Computador Socket\n'),
```

Comment:
Não vi motivos pra ter um modo de jogo computar online se você já tem o local

Reply:
O motivo é que pode utilizar outros clientes, além do console do prolog. Esse modo não vai aumentar a complexidade, pq a mesma implementação do socket pro multiplayer vai funcionar nesse

Reply:
Ainda assim não vejo muito motivo, já que pra a pessoa utilizar outro cliente ela teria que fazer uma espécie de conector com a outra API. Enfim, se for só por isso o pr tá tranquilo pra mim.